script/Ciel/Day 1/01_01_CIEL01_2_3.txt

```diff
@@ -1067,7 +1067,7 @@ What made you suddenly think that I'm a transfer student?"
 [sha:70bdc8094db70f9f55cfad28815c92703df90b6e]{
 -- Page 140, Offset 828.
 -- 今までは親戚の家にいたんですけど、今日からもとの家に戻るだけなんですよ」
-I've been living with my relatives up til now, but from today on I'll be back at my old house."
+I've been living with my relatives up till now, but from today on I'll be back at my old house."
```

Comment:
Personally pro- plain `til`, at least in casual (spoken) contexts. 
https://www.dictionary.com/e/til-or-till/

Reply:
It's from mw again, Which dictionary should we follow then?
![image](https://github.com/Tsukihimates/Tsukihime-Translation/assets/90473809/2da6035e-3603-49a5-a86d-4823c0ac8af2)

Reply:
this is casual speech, it's not going to exactly match the dictionary. English, no rules, etc.

Reply:
alright i will add the comma too